templates/template-spring-boot-maven-mysql/files/.platform/local/README.txt

```diff
@@ -0,0 +1,8 @@
+.platform/local
```

Comment:
Need to ignore everything in .platform/local so it doesn't push to the template repos.